annual-calendar-creator/README.md

```diff
@@ -0,0 +1,52 @@
+# Annual Calendar Creator
+
+Annual Calendar Creator is a Lucid application that re-organizes shapes and elements on your documents into circles.
```

Comment:
> that re-organizes shapes and elements on your documents into circles.
I'm not 100% certain, but I don't think that's what this extension does :slightly_smiling_face: (similar phrasing exists in the `## Terms of use` section below.

This file seems very similar to the `annual-calendar-creator/editorextensions/README.md` one. Are both needed? Can one just be a link to the other so you don't have to repeat yourself?

Reply:
Both are not needed, I'll remove one of them.